iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/bucket/DocOrgRelationAffOrgJoiner.java

```diff
@@ -0,0 +1,48 @@
+package eu.dnetlib.iis.wf.affmatching.bucket;
+
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+
+import eu.dnetlib.iis.wf.affmatching.bucket.projectorg.model.AffMatchDocumentOrganization;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchAffiliation;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchOrganization;
+import scala.Tuple2;
+
+/**
+ * Implementation of {@link AffOrgJoiner} that joins {@link AffMatchAffiliation} with {@link AffMatchOrganization}
+ * based on document-organization relations.
+ * 
+ * @author madryk
+ */
+public class DocOrgRelationAffOrgJoiner implements AffOrgJoiner {
+
+    private static final long serialVersionUID = 1L;
+
+
+    //------------------------ LOGIC --------------------------
+
+    /**
+     * Joins the given affiliations with organizations based on document-organization relations.<br />
```

Comment:
I was thinking more about explaining how those document-organization auxiliary relations are helpful in matching affiliations with organizations. In a big picture. 

You're probably right, this description block might not be the best place to clarify on this subject since it works on lower level, but it will be nice to find such place to make logic behind this clear.